examples/core/quad/main.rs

```diff
@@ -117,19 +117,11 @@ fn main() {
     // dx12 runtime shader compilation
     #[cfg(feature = "dx12")]
     let vs_module = device
-        .create_shader_module_from_source(
-            pso::Stage::Vertex,
-            "vs_main",
-            "main",
-            include_bytes!("shader/quad.hlsl"))
+        .create_shader_module(include_bytes!("data/vert.spv"))
```

Comment:
Can we switch the vulkan path to also get SPIRV for consistency, merging them here?